src/qmplay2/Functions.cpp

```diff
@@ -612,3 +633,19 @@ bool Functions::wrapMouse(QWidget *widget, QPoint &mousePos, int margin)
 
 	return doWrap;
 }
+
+QStringList Functions::getProfiles()
+{
+	QStringList profiles;
+
+	QDirIterator it(QMPlay2Core.getSettingsDir() + "Profiles/", QStringList() << "QMPlay2.ini", QDir::Files, QDirIterator::Subdirectories);
```

Comment:
Filtering `QMPlay2.ini` shouldn't be necessary, just show only available directories. This will prevent creating an empty file. Empty directory should be a clean, new profile. QMPlay2 will automatically creates `ini` files :)